ray-curator/ray_curator/tutorials/text/datasets_download_extract_tutorial.ipynb

```diff
@@ -0,0 +1,651 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d09fc8a0-9024-49a6-91e8-1aa851535ea1",
+   "metadata": {},
+   "source": [
+    "# NeMo Curator Download and Extract Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4913b883-3b62-49c3-b08f-426c9dc43047",
+   "metadata": {},
+   "source": [
+    "## About This Notebook\n",
+    "NeMo Curator has pre-built download and extract piplines for **Common Crawl**, **Wikipedia** and **ArXiv** datasets. In this tutorial, we will introduce how to execute these pipelines.\n",
+    "\n",
+    "#### The tutorial is tested in below system:\n",
+    "* OS: Ubuntu 24.04.2 LTS\n",
+    "* RAM: 128 GB\n",
+    "* Software environment: the notebook is recommended to be launched in the NeMo Curator's docker container:\n",
+    "```bash\n",
+    "git clone https://github.com/NVIDIA-NeMo/Curator.git\n",
+    "cd Curator\n",
+    "docker build -f ray-curator/Dockerfile -t nemo-curator-ray .\n",
+    "docker run -it --name nemo-curator-ray -p 8888:8888 nemo-curator-ray bash\n",
+    "cd /opt/ray_curator/tutorials/text\n",
+    "jupyter-lab --no-browser --allow-root --ip 0.0.0.0\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c90fe3d",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "`wget` will be needed in the turorial for data downloading."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d06f3ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!apt update\n",
+    "!apt install -y wget"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5967b66a-94ba-4392-a1bf-58ae3b3add9e",
+   "metadata": {},
+   "source": [
+    "## Common Crawl Download and Extract\n",
+    "\n",
+    "### About Common Crawl Dataset\n",
+    "The [Common Crawl](https://commoncrawl.org/) dataset is a massive, openly available archive of web data collected by the nonprofit Common Crawl organization since 2008. It consists of petabytes of raw web page data, metadata extracts, and text extracts, providing a valuable resource for anyone interested in large-scale web analysis or AI research. Common Crawl currently stores the crawl data using the [Web ARChive (WARC)](https://en.wikipedia.org/wiki/WARC_(file_format)) format. In this section, we will explore how to use NeMo Curator to download the Common Crawl raw data (WARC files) and extract the text from the it. "
```

Comment:
Hmm I think this link is broken, because if the parentheses in the link for `(file_format)`. Can you please fix?